CMakeLists.txt

```diff
@@ -51,6 +51,14 @@ configure_file(
 )
 install(FILES ${CCF_DIR}/include/ccf/version.h DESTINATION include/ccf)
 
+# TODO: Not very elegant...
```

Comment:
This is to bake `cchost_config.json` as a single string variable in the `cchost` binary. I can't think of anything more minimal than this, but it's not very elegant. Suggestions welcome!

Reply:
Can we use a raw string literal, rather than manual escaping? See eg `j_record_public_in` in `logging_schema.h` for an example. So `config_schema.h.in` would contain something like `host_config_schema = R"!!!(@HOST_CONFIG_SCHEMA@)!!!";`